doc/connectivity/networking/api/lwm2m.rst

```diff
@@ -483,7 +483,7 @@ The events are prefixed with ``LWM2M_RD_CLIENT_EVENT_``.
    * - 6
      - REG_UPDATE_FAILURE
      - Registration update failed.
-       Occurs if there is a timeout during registration update.
+       Occurs if there is a timeout during registration or registration update.
```

Comment:
I think my intentions were misunderstood here (sorry for not being clear enough). My point was, that we should be consistent with the events for Regsiter/Update/Deregister operations:
* either handle both timeout and error reply as the same kind of event for all operations (REGISTRATION_FAILURE/UPDATE_FAILURE/DEREGISTER_FAILURE) - that's how it's done right now,
* or introduce a new event for each operation which would allow to differentiate between timeout and error response (REGISTRATION_TIMEOUT/UPDATE_TIMEOUT/DEREGISTER_TIMEOUT).

If, from application perspective, it's important to differentiate between timeout and error reply, I think we should do the latter. But I'm strongly in favour of introducing new, specific events, instead of hiding the timeout event under one of the existing ones (which IMO could be more confusing than the current situation).

Reply:
OK. I create a new Event. I was wondering that we not break old application which missing a new event type.